### ISSUE TITLE
"Orientation Estimation Network" refer to which paper?

### ISSUE THREAD

**Issue by xue1liu2** [sunjieee/MobileNets-Tensorflow#1]
There are a lot of "Orientation Estimation Network" related paper.
Could you give its all name?

which dataset you are using, could you give the dataset's name?

thanks anyway.

**Comment by sunjieee** [comment]
1. Wu, F., Hu, P., and Kong, D. Flip-Rotate-Pooling Convolution and Split
Dropout on Convolution Neural Networks for Image Classification. arXiv preprint
arXiv:1507.08754, 2015.
2. Marcos, D., Volpi, M., and Tuia, D. Learning rotation invariant convolutional filters
for texture classification. arXiv preprint arXiv:1604.06720, 2016.
3. Zhou, Y., Ye, Q., Qiu, Q., and Jiao, J. Oriented Response Networks. arXiv preprint
arXiv:1701.01833, 2017.
4. Marcos, D., Volpi, M., Komodakis, N., and Tuia, D. Rotation equivariant vector
field networks. arXiv preprint arXiv:1612.09346, 2016.
5. Cohen, T. S., and Welling, M. Steerable CNNs. arXiv preprint arXiv:1612.08498, 2016.
6. Worrall, D. E., Garbin, S. J., Turmukhambetov, D., and Brostow, G. J. Har-
monic Networks: Deep Translation and Rotation Equivariance. arXiv preprint
arXiv:1612.04642, 2016.
